go.mod

```diff
@@ -1,23 +1,23 @@
 module sigs.k8s.io/gateway-api
 
-go 1.21
+go 1.22.2
 
 require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
 	github.com/miekg/dns v1.1.58
 	github.com/stretchr/testify v1.9.0
-	golang.org/x/net v0.20.0
+	golang.org/x/net v0.23.0
 	golang.org/x/sync v0.6.0
 	google.golang.org/grpc v1.61.0
 	google.golang.org/protobuf v1.33.0
-	k8s.io/api v0.29.3
-	k8s.io/apiextensions-apiserver v0.29.3
-	k8s.io/apimachinery v0.29.3
-	k8s.io/client-go v0.29.3
-	k8s.io/code-generator v0.29.3
-	k8s.io/kube-openapi v0.0.0-20240105020646-a37d4de58910
+	k8s.io/api v0.30.0
+	k8s.io/apiextensions-apiserver v0.30.0
+	k8s.io/apimachinery v0.30.0
+	k8s.io/client-go v0.30.0
+	k8s.io/code-generator v0.30.0
+	k8s.io/kube-openapi v0.0.0-20240228011516-70dd3763d340
 	k8s.io/utils v0.0.0-20240102154912-e7106e64919e
-	sigs.k8s.io/controller-runtime v0.16.3
+	sigs.k8s.io/controller-runtime v0.17.3
```

Comment:
surprised this works since they don't support 1.30 yet and there were incompatible changes.. I guess we must not import some of those packages

Reply:
Bumped back to v0.17.0 to allow dependent implementations to pick up newer changes from controller-runtime - no changes in either case from perspective of Gateway API directly.